### DISCUSSION TITLE
Can I extend the features for my example?

### DISCUSSION THREAD

**Discussion by gutentag2012** [realworld-apps/realworld#1583] [Q&A] [answered]
Hello there, would extending the list of features listed in the specs be acceptable for a fullstack implementation?

I wanted to create a bigger example application that tries to improve authentication and adds new features that can be controlled via feature flags. Adding observability through Open Telemetry would also be one of the goals for this example.
There are a few more additional features that I would like to take a look at.

The idea is that all the specs are fulfilled and all other features are hidden behind feature flags to be configured.

So the question is if such an extended project would be acceptable.

**Comment by geromegrignon** [comment, marked as answer]
Hello,

You can surely create it but the point of the project is to help people comparing implementations so they must follow specs to be listed.

**Reply by gutentag2012** [reply]
Hey there,

thanks for the answer, I will see how I manage it, if it gets too complex for an easy comparison I could just move the extra logic to a separate branch an reference it in the README.
I want to build everything around the specs and just sprinkle in some other features to demonstrate how a large distributed system could work.

The main application will be built with NextJS in the Frontend, NestJS in the backend and maybe also a React Native App all together in one codebase.
So overall this could take a while until I can submit something, but then I hope it can get accepted 😄
